### ISSUE TITLE
Dev test: Balloons and Static Electricity

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#112]
@emily-phet @terracoda @ariel-phet @phet-steele, Balloons and Static Electricity 1.4.0-dev.21 is ready for dev testing. This version includes accessibility features that are compatible with a screen reader.

Before beginning, familiarize yourself with how a screen reader works. Here is a page with information about PhET's supported screen readers and documentation about how to use them:
[Screen reader intro](https://www.colorado.edu/physics/phet/dev/html/jg-tests/reader-intro.html)

**[Link to sim](https://www.colorado.edu/physics/phet/dev/html/balloons-and-static-electricity/1.4.0-dev.21/phet/balloons-and-static-electricity_all_phet_debug.html?accessibility)**

The above sim has labels and/or descriptions. They can be "seen" here:
**[Link to a11y view](https://www.colorado.edu/physics/phet/dev/html/balloons-and-static-electricity/1.4.0-dev.21/phet/balloons-and-static-electricity_a11y_view.html)**

This view should assist in seeing what is available to a screen reader user. While testing,  please make sure that all descriptions and alerts accurately describe the state of the simulation. All content in the a11y-view should be readable with a screen reader when one is active.

PhET supports the following platforms for accessibility so please test these:
- [x] JAWS with latest Windows, latest Firefox
- [x] NVDA with latest Windows, latest Firefox
- [x] VoiceOver with latest macOS, latest Safari

In addition to the above,  please also do some normal dev testing of this sim without a screen reader.

If any new issues are found, please note them in https://github.com/phetsims/balloons-and-static-electricity/issues and reference this issue. 

**Critical screen reader information**
- JAWS might complain about Firefox Quantum. If that is the case, please test with https://www.mozilla.org/en-US/firefox/organizations/
- The screen reader **must** be turned on and reading before the browser is open for it to work. See https://github.com/phetsims/a11y-research/issues/90
 - JAWS might not automatically switch to forms mode for some controls (like sliders and radio buttons). In this case, pressing "enter" should switch to forms mode. For example, see https://github.com/phetsims/resistance-in-a-wire/issues/138
 - The screen reader might focus the last UI component that had focus on sim load, typically on refresh. This is a "feature". For example, see https://github.com/phetsims/resistance-in-a-wire/issues/139

@ariel-phet please assign and prioritize

**Comment by jessegreenberg** [comment]
>JAWS might not automatically switch to forms mode for some controls (like sliders and radio buttons). In this case, pressing "enter" should switch to forms mode.

@phet-steele mentioned that there is not enough context to understand this. I will add information about this to the "Screen Reader Intro".

**Comment by phet-steele** [comment]
@jessegreenberg it might be useful to link the github issues that led to us writing the "critical information". So, what issue caused us to realize the JAWS may not switch to forms mode, etc.?

**Comment by jessegreenberg** [comment]
Good idea @phet-steele, I added examples to the screen reader quirks in the issue  and a11y test templates.

**Comment by lmulhall-phet** [comment]
QA is done.

**Comment by jessegreenberg** [comment]
Excellent, thanks for the hard work @lmulhall-phet and team, closing this issue.
